packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -4,7 +4,11 @@
 			<template v-if="!node.inputs[0].value"> Attach a model configuration </template>
 		</tera-operator-placeholder>
 		<template v-if="node.inputs[0].value">
-			<tera-progress-spinner v-if="showSpinner" :font-size="2" is-centered style="height: 100%" />
+			<div v-if="showSpinner">
+				<p>{{ node.state.currentProgress }}% of maximum iterations complete</p>
+				<tera-progress-spinner :font-size="2" is-centered style="height: 100%" />
+			</div>
```

Comment:
Your usage of `tera-progress-spinner` is a bit odd since it is slotted for a message but you are putting the message outside I think?

Can we do something like this so the usage is more idiomatic across the app?
```
   <tera-progress-spinner v-if="inProgressCalibrationId" :font-size="2" is-centered style="height: 100%">
     <div>{{ props.node.state.currentProgress }}%</div>
   </tera-progress-spinner>
```

Reply:
Good catch thank you, will do